ingest/sqs/service/sqs_streaming_service.go

```diff
@@ -197,6 +188,22 @@ func (s *sqsStreamingService) processBlock(ctx sdk.Context) error {
 	return s.sqsIngester.ProcessChangedBlockData(ctx, changedBlockPools)
 }
 
+func (s *sqsStreamingService) processBlockChangeSet() error {
+	if s.changeSet == nil {
+		return nil
+	}
+
+	for _, kv := range s.changeSet {
+		for _, listener := range s.writeListeners[s.storeKeyMap[kv.StoreKey]] {
+			if err := listener.OnWrite(s.storeKeyMap[kv.StoreKey], kv.Key, kv.Value, kv.Delete); err != nil {
+				return err
+			}
+		}
+	}
+
+	return nil
+}
```

Comment:
I wonder how this affects block time processing. We should create an issue and track validating that the performance hit is acceptable relative to the expected lowered block time

Reply:
Added this to the list of things to test